src/components/views/createplayer/CreatePlayer.tsx

```diff
@@ -20,13 +21,19 @@ const CreatePlayer = () => {
   const changeName = (e) => {
     setName(e.target.value)
   }
+  const redirectLink = useQuery().get("redirectTo")
+  console.log(JSON.stringify(redirectLink))
 
   const { me, loading, createPlayer, startPollPlayers } = usePlayers()
 
   const submit = async (e) => {
     e.preventDefault()
     await createPlayer(name)
-    navigate(Paths.LOBBY)
+    if (redirectLink === null) {
```

Comment:
```suggestion
    if (!redirectLink) {
```
The link may also be an empty string, undefined, 0, false. (Yes, javascript at its best)
In these cases we don't want to navigate there.